### DISCUSSION TITLE
创建新的库的时候，如何同步设置和插件？

### DISCUSSION THREAD

**Discussion by leozjr** [obsidianzh/forum#54] [Q&A常见问题]
创建了一个新库之后，设置都变成默认的了，插件也没了，发现可以直接复制旧的.obsidian文件夹替换，有没有有更简便的方法？

**Comment by makerjackie** [comment]
好像没有....复制 .obsidian 是最快的方法

**Comment by hejie13250** [comment]
这里有完美解决方法https://thoughts.teambition.com/sharespace/60612eb785c2c10046588f89/docs/6062e14e85c2c100465892d7
